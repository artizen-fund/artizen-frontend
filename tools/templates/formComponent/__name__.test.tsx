import renderer from 'react-test-renderer'
import __name__ from './'

describe('__name__', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<__name__ />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
