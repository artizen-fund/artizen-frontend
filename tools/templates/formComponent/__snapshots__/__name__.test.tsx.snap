// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__name__ renders correctly 1`] = `
<div
  className="sc-64f7a6f3-0"
>
  <div
    className="vertical-layout"
    hidden={false}
  >
    <div
      className="vertical-layout-item"
    >
      <div
        className="sc-2f9d5146-0 imlAYL"
        id="#/properties/someString"
      >
        <div
          className="sc-233fee17-0 cRUHBc"
        >
          <input
            defaultValue="Some default value"
            minLength={2}
            onBlur={[Function]}
            onChange={[Function]}
            placeholder=" "
            required={false}
            type="text"
          />
          <label
            className="sc-ffdb2e6c-0 dXVxRc"
          >
            Tell us a thing.
          </label>
        </div>
        <div
          className="sc-93ba83c0-0 bAVzHI"
        />
      </div>
    </div>
    <div
      className="vertical-layout-item"
    >
      <div
        className="sc-2f9d5146-0 eQAzBx"
        id="#/properties/someEmail"
      >
        <div
          className="sc-233fee17-0 cRUHBc"
        >
          <input
            onBlur={[Function]}
            onChange={[Function]}
            placeholder="your@email.com"
            required={true}
            type="text"
          />
          <label
            className="sc-ffdb2e6c-0 dXVxRc"
          >
            Email Address
             *
          </label>
        </div>
        <div
          className="sc-93ba83c0-0 bAVzHI"
        />
      </div>
    </div>
    <div
      className="vertical-layout-item"
    >
      <div
        className="sc-2f9d5146-0 cKQxBV"
        id="#/properties/someNumber"
      >
        <div
          className="sc-233fee17-0 cRUHBc"
        >
          <input
            className="noData"
            onBlur={[Function]}
            onChange={[Function]}
            placeholder=" "
            required={false}
            step="any"
            type="number"
          />
          <label
            className="sc-ffdb2e6c-0 dXVxRc"
          >
            What is your lucky number?
          </label>
        </div>
        <div
          className="sc-93ba83c0-0 bAVzHI"
        />
      </div>
    </div>
    <div
      className="vertical-layout-item"
    >
      <label
        className="sc-dd35d48e-4 hfvlkX"
        id="#/properties/someBoolean"
      >
        <div
          className="sc-dd35d48e-0 lmHjtw"
        >
          <input
            checked={true}
            className="sc-dd35d48e-1 clTMzK"
            disabled={false}
            onChange={[Function]}
            required={true}
            type="checkbox"
          />
          <span
            className="sc-dd35d48e-2 Legqi"
          />
        </div>
        <span
          className="sc-dd35d48e-3 kiyWfS"
        >
          Do you like to click checkmarks?
        </span>
      </label>
    </div>
  </div>
  <button
    className="sc-a6b3add2-2 ifpeQA undefined "
    onClick={[Function]}
  >
    <span>
      Submit
    </span>
  </button>
</div>
`;
